cmd/util/applicationset_test.go

```diff
@@ -31,10 +31,10 @@ spec:
 `
 
 func TestReadAppSet(t *testing.T) {
-	appsets := []*argoprojiov1alpha1.ApplicationSet{}
-	err := readAppset([]byte(appSet), &appsets)
+	var appset []*argoprojiov1alpha1.ApplicationSet
```

Comment:
I think `appSets` makes more sense. This is a slice of appsets.

Reply:
Empty slice declaration using a literal

Reply:
Yep, the change of the declaration is fine. I think changing the variable name was incorrect though.

Reply:
@crenshaw-dev done